ReactiveCocoa/UIKit/UITextView.swift

```diff
@@ -7,6 +7,11 @@ extension Reactive where Base: UITextView {
 	public var text: BindingTarget<String> {
```

Comment:
Makes me wonder if this should be `String?` instead, or instead we hide the nullability and consistently use an empty string in our APIs.